db/migrate/20220618072918_create_activities.rb

```diff
@@ -1,7 +1,7 @@
 class CreateActivities < ActiveRecord::Migration[6.1]
   def change
     create_table :activities do |t|
-      t.string :name
+      t.string :title
```

Comment:
I changed this to :title instead of :name, i believe its a better table_name/representation for the Activities Model

Reply:
Yes, I agree, will avoid confusion